### ISSUE TITLE
Documentation for End-to-End RE task

### ISSUE THREAD

**Issue by yamini7595** [smitkiri/ehr-relation-extraction#9]
Is there any written documentation explaining the details of the end-to-end RE task?

**Comment by smitkiri** [comment]
I don't have a code-walkthrough of the end-to-end task, but the general idea is to get the entities from an NER model and pass it as an input to the RE model. You can get a good understanding of the process if you go through the `fast_api.py` and `predict.py` files.

**Comment by smitkiri** [comment]
I've added more comments in the `predict.py` file in the torchserve branch in case the code was hard to follow along.

**Comment by yamini7595** [comment]
Thank you!
